__tests__/news.test.js

```diff
@@ -49,8 +49,53 @@ describe.only("GET/api/articles/:article_id", () => {
     return request(app)
       .get("/api/articles/9999")
       .expect(400)
+      .then((result) => {});
+  });
+});
+
+describe("PATCH/api/articles/:article_id", () => {
+  test("responds with changed article", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 1 })
+      .expect(200)
+      .then((result) => {
+        expect(result.body.article).toBeInstanceOf(Object);
+        expect(result.body.article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+        });
+        expect(result.body.article.votes).toBe(101);
+      });
+  });
+  test("responds with changed article", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: -10 })
+      .expect(200)
       .then((result) => {
-        console.log("THIS IS MY CHANGE");
+        expect(result.body.article).toBeInstanceOf(Object);
+        expect(result.body.article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+        });
+        expect(result.body.article.votes).toBe(90);
       });
   });
+  test("responds with correct error message", () => {
+    return request(app)
+      .patch("/api/articles/9999")
+      .expect(400)
```

Comment:
This should still be a 404 and you need to still send a body. A 400 would be a 'bad request' - how do you perform a bad request with a patch?